pke/kyber/internal/common/poly_test.go

```diff
@@ -26,7 +27,11 @@ func TestDecompressMessage(t *testing.T) {
 	var m, m2 [PlaintextSize]byte
 	var p Poly
 	for i := 0; i < 1000; i++ {
-		_, _ = cryptoRand.Read(m[:])
+		_, err := rand.Read(m[:])
```

Comment:
nit: I would handle the first output handle paraemter.

Reply:
resolved